gwf/core.py

```diff
@@ -238,6 +230,41 @@ def target(self, name, inputs, outputs, **options):
         self._add_target(new_target)
         return new_target
 
+    def target_from_template(self, name, template, **options):
+        """Create a target and add it to the :class:`gwf.Workflow`.
+
+        This is syntactic sugar for creating a new :class:`~gwf.Target` and
+        adding it to the workflow. The target is also returned from the method
+        so that the user can directly manipulate it, if necessary. For example,
```

Comment:
Remove "For example, ...". It's confusing since that's not what the example is showing.